app/src/main/java/com/framgia/music_49/screen/home/ApdaterGenre.java

```diff
@@ -0,0 +1,68 @@
+package com.framgia.music_49.screen.home;
+
+import android.content.Context;
+import android.support.annotation.NonNull;
+import android.support.v7.widget.RecyclerView;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+import android.widget.ImageView;
+import com.bumptech.glide.Glide;
+
+import com.framgia.music_49.data.model.Genre;
+import com.framgia.music_49.utils.ItemClickListener;
+import com.framgia_music_49.R;
+import java.util.List;
+
+public class ApdaterGenre extends RecyclerView.Adapter<ApdaterGenre.ItemHolder> {
+    private Context mContext;
+    private List<Genre> mGenres;
+    private ItemClickListener mItemClickListener;
+
+    public ApdaterGenre(Context context, List<Genre> genres, ItemClickListener itemClickListener) {
+        mContext = context;
+        mGenres = genres;
+        mItemClickListener = itemClickListener;
+    }
+
+    @NonNull
+    @Override
+    public ItemHolder onCreateViewHolder(@NonNull ViewGroup viewGroup, int i) {
+        View view = LayoutInflater.from(viewGroup.getContext())
+                .inflate(R.layout.item_view_genre, viewGroup, false);
+        return new ItemHolder(view, mItemClickListener);
+    }
+
+    @Override
+    public void onBindViewHolder(@NonNull ItemHolder viewHolder, int position) {
+        viewHolder.binData(mGenres.get(position));
+    }
+
+    @Override
+    public int getItemCount() {
+        return mGenres != null ? mGenres.size() : 0;
+    }
+
+    public class ItemHolder extends RecyclerView.ViewHolder implements View.OnClickListener {
```

Comment:
https://stackoverflow.com/questions/13516835/why-in-viewholder-pattern-should-the-viewholder-class-be-static